### ISSUE TITLE
Closing teaspeak always takes 30s

### ISSUE THREAD

**Issue by HAMMERxPL** [TeaSpeak/TeaSpeak#478]
[2018-10-16 16:17:06] [CRITICAL]   GEN | Could not shutdown server within 30 seconds! (Hangup!)
[2018-10-16 16:17:06] [CRITICAL]   GEN | Killing server!
> pure virtual method called
terminate called without an active exception
pure virtual method called
terminate called recursively

**Comment by DanielAlves123** [comment]
@WolverinDEV still exists.

**Comment by WolverinDEV** [comment]
Of cause sometimes, but it isnt the general procedure

**Comment by DanielAlves123** [comment]
@WolverinDEV still exists.

**Comment by DanielAlves123** [comment]
@WolverinDEV can you look into that?

**Comment by WolverinDEV** [comment]
As Ive already said.
It's not a general issue. 
It just happens sometimes when something hangs up. Than the 30s timeout triggers

**Comment by DanielAlves123** [comment]
@WolverinDEV not true. It ALWAYS takes 30s to close, even if nothing crashed or hang up.
